src/js/zoneUtils.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 exports = module.exports = {
-    STATUS_NOT_STARTED: 1,
-    STATUS_FINISHED: 2,
+    STATUS_NOT_STARTED: 0,
```

Comment:
Wondering if we should just change these to booleans. Since this change effectively makes them as such.